### DISCUSSION TITLE
Directus flow (read data), filter_count

### DISCUSSION THREAD

**Discussion by lyneal** [directus/directus#16666] [Archive]
Hello,

In order to lighten my front-end application I try to internalize as much as possible the work within directus, in particular by creating flows to load my data using "read data".

In the query part everything works correctly except "meta" which does not allow me to retrieve the filter_count.

Could you please tell me if this is a bug or how I can retrieve this data?

I thank you in advance !

**Comment by azrikahar** [comment]
I believe it is currently not possible (but isn't technically a bug) because "meta" isn't implemented in Flow CRUD operations.

meta works via the API as it (`metaService`) is, separately, invoked in the controller after items are read:

https://github.com/directus/directus/blob/d6b8e5963d955769cfe2a721014ca17191d8850c/api/src/controllers/items.ts#L73-L86

but the Read Data operation currently only reads data without using the meta service:

https://github.com/directus/directus/blob/d6b8e5963d955769cfe2a721014ca17191d8850c/api/src/operations/item-read/index.ts#L44-L56

**Reply by lyneal** [reply]
I have a workaround which is to use the aggregation system.

I don't know if it's possible to get the aggregation on one side and the data on the other side in one query, but if not, I simply make several stream spots and then combine them with the "Transform payload".
